codegen/header.go

```diff
@@ -18,15 +18,18 @@ func Header(title, pack string, imports []*ImportSpec) *SectionTemplate {
 	}
 }
 
-// AddImport adds an import to a section template that was generated with
+// AddImports adds imports to a section template that was generated with
 // Header.
-func AddImport(section *SectionTemplate, imprt *ImportSpec) {
+func AddImports(section *SectionTemplate, imprts ...*ImportSpec) {
```

Comment:
AddImport is used in quite a few places in goa v2 plugins - https://github.com/goadesign/plugins/blob/master/cors/generate.go#L86